pom.xml

```diff
@@ -17,20 +17,50 @@
     <parent>
         <artifactId>jberet-parent</artifactId>
         <groupId>org.jberet</groupId>
-        <version>1.4.0.Alpha-SNAPSHOT</version>
+        <version>2.0.3.Final</version>
     </parent>
 
     <artifactId>jberet-jpa-repository</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>jakarta.xml.bind</groupId>
+            <artifactId>jakarta.xml.bind-api</artifactId>
+        </dependency>
         <dependency>
             <groupId>org.jberet</groupId>
             <artifactId>jberet-core</artifactId>
             <version>${project.version}</version>
         </dependency>
```

Comment:
any jberet-* artifacts should have their own version, not using the version of this project, since they will be different in the future. Can you add a property for jberet-core and reference here?